src/marshaller/HipieDDL.js

```diff
@@ -1489,12 +1536,20 @@
                     return rowValue.localeCompare(value) >= 0;
                 }
                 return rowValue >= value;
+            case "!=":
+            case "notequals":
```

Comment:
I switched "notequals" to "!=" in the latest changes to HIPIE for range filtering. You shouldn't encounter any "notequals" occurrances.